src/gax/src/client_builder.rs

```diff
@@ -423,6 +437,7 @@ pub mod internal {
         pub retry_throttler: SharedRetryThrottler,
         pub polling_error_policy: Option<Arc<dyn PollingErrorPolicy>>,
         pub polling_backoff_policy: Option<Arc<dyn PollingBackoffPolicy>>,
+        pub instrumentation: Option<InstrumentationOptions>,
```

Comment:
@dbolduc this type is non-exhaustive, so adding a new field is non-breaking. That is intentional to permit this type of addition.